code/modules/ai/ai_holder_subtypes/slime_xenobio_ai.dm

```diff
@@ -126,7 +126,7 @@
 				LAZYDISTINCTADD(yummies, M)
 	if (!LAZYLEN(yummies))
 		return
-	var/atom/yummy = pick(yummies)
+	var/atom/yummy
 	var/closest_distance = get_dist(holder, yummy)

```

Comment:
What's the distance between `holder` and `NULL`?

Reply:
gah, I missed that as part of putting in the other commit. it still seems to *work* in the code, so I'm guessing it's something serviceable, but I'll change it to something better regardless

Reply:
Not sure if this actually applies given it seemed to work, but per the byond reference:
> At this time, get_dist() never returns a value greater than 127.
For a distance in pixels, use bounds_dist().
get_dist() will return -1 in error conditions, such as when Loc1 and Loc2 are the same object.